How-to-access-Oppia-webpages.md

```diff
@@ -1,237 +1,336 @@
+
+### Admin page
+
+The admin page is a page accessible only to super-administrators. It has many functionalities including changing permissions, configuration values, and running jobs. Certain webpages cannot be accessed unless the current user has the required permissions.
+
+1. Log in as a super-admin.
 
-![Sign-In page](https://user-images.githubusercontent.com/16653571/41500954-e88a0262-71b8-11e8-9cac-456fb46782c1.png)
+2. Go to the profile menu and click on the "Admin Page" link.
 
-## Webpage Guide
+   ![Admin Panel Link](https://user-images.githubusercontent.com/16653571/41501009-e04e9a76-71b9-11e8-958e-985f5bc7122a.png)
 
-**Admin Page**
+#### Assign roles
 
-The Admin page is a page exclusively for developers. It has many functionalities including changing permissions, configuration values, and running jobs. Certain webpages cannot be accessed unless the current user does not have the required permissions. 
+1. Navigate to the Admin page.
 
-1. Login into the Oppia as an administrator
+2. Click the "ROLES" tab.
 
-![Sign-In page](https://user-images.githubusercontent.com/16653571/41500954-e88a0262-71b8-11e8-9cac-456fb46782c1.png)
+   ![Admin Role Tab](https://user-images.githubusercontent.com/16653571/41501684-a543b22e-71c6-11e8-883c-cec35e32535e.png)
 
-2. Go to the profile menu and click on the "Admin" link
+3. Change the role of your username to the desired role, "collection editor" in this example.
 
-![Admin Panel Link](https://user-images.githubusercontent.com/16653571/41501009-e04e9a76-71b9-11e8-958e-985f5bc7122a.png)
+   ![assignrole](https://user-images.githubusercontent.com/16653571/41503346-ff754c9e-71ee-11e8-9b72-9e16dae46418.png)
 
-**About Page**
+### About page
 
-The About page provides a brief overview of Oppia. It details the goals of the Oppia organization, credits its contributors, and provides links to guides and tutorials on how to further explore Oppia
+The About page provides a brief overview of Oppia. It details the goals of the Oppia organization, credits its contributors, and provides links to guides and tutorials on how to further explore Oppia.
 
-1. Click the About Menu button on the top right navigation bar
+1. Click the "About Menu" button on the top right navigation bar.
 
-2. Click on the “About Oppia” link
+2. Click on the "About Oppia" link.
 
-![AboutPageLink](images/Webpage-Guide/aboutPageLink.png)
+   ![AboutPageLink](images/Webpage-Guide/aboutPageLink.png)
 
-**Collection_editor**
+### Collection editor page
 
-The Collection editor page allows users to create collections in Oppia. The collection editor page can only be accessed by users with the role _COLLECTION_EDITOR_
+The Collection editor page allows users to create collections, which group explorations together. The collection editor page can only be accessed by users with the "collection editor" role.
 
-1. Navigate to the Admin page
-2. Click the **ROLES** tab
+1. Log in as a super-admin and change your user's role to "collection editor."
```

Comment:
We are going to have a multi-role system where user have-to-have the full user role, over here it will be "add role".

Reply:
Like in my response comment above, I think we should only update the docs after the code has changed

Reply:
DItto as above!

Reply:
done